libraries/SITL/SIM_Sailboat.cpp

```diff
@@ -291,6 +291,12 @@ void Sailboat::update(const struct sitl_input &input)
     // add in accel due to direction change
     accel_body.y += radians(yaw_rate) * speed;
 
+    if (!input.initialised()) {
+        // outputs are not yet initialised, prevent unintended
+        // movement
+        accel_body.zero();
```

Comment:
This should zero `throttle_force` before line 288. That way sail force and hull drag will still apply.